src/utils/builder.ts

```diff
@@ -35,28 +35,73 @@ export function createNewPathway(name: string, description: string, pathwayId?:
   };
 }
 
+interface AddCriteriaSourceInterface {
+  updated: boolean;
+  newPathway: Pathway;
+}
+
+export function addCriteriaSourceToPathway(
```

Comment:
I feel like `updateCriteriaSources` would be a more meaningful name. ( or `updatePathwayCriteriaSources`?)

Reply:
Changed to `updatePathwayCriteriaSources`